### DISCUSSION TITLE
Creatring custom "Multiple assignments" drawers

### DISCUSSION THREAD

**Discussion by hexdump2002** [wojtryb/Shortcut-Composer#54] [Ideas]
Just like default photoshop, it would be really nice if we could create custom "Multiple assignments" groups beside the built-in ones Opacity and Selection tools. For example, I would like to manage the Rectangular, free hand and poligonal selection tools under the same shortcut. Another example is having the fill bucket and gradient tool selected under the same shortcut too.

By the way, thanks for this awesome add-on :). Congratulations!

**Comment by wojtryb** [comment]
Hi,

on github you can switch from "main" to "feature/gui-action-editing", then download the .zip again, and import it from krita. It adds new tab in `Configure Shortcut Composer` dialog where you can configure existing actions.

If you want to create your own actions you can:
1. Download VsCode https://code.visualstudio.com/
2. In krita go to: `Settings > Manage Resources > Open Resource Folder`
4. Go to `./pykrita/` and drag&drop `shortcut-composer` onto VsCode
5. Open `actions.py` and `actions.action` files. Let VsCode install the python extension.

You can copy paste any of the multiple assignment actions and fill it with your names and values. VsCode will give you auto-completion and display my hints when you hover over parts of code. Remember to edit both files (definition and implementation).

Creating these from inside krita may come at some point, but it's not the highest priority right now.
